.gitignore

```diff
@@ -55,3 +55,6 @@ dolphinscheduler-master/logs
 dolphinscheduler-api/logs
 __pycache__
 ds_schema_check_test
+.env.txt
+venv
+myenv
```

Comment:
Please remove this

Reply:
Sure. Sir. Thank you. I will also try to find a way to exclude these log4j-api and log4j-to-slf4j. I was looking on the same too.

Reply:
We need to find out where these two dependencies are imported, and exclude these, otherwise the ci will failed.

Reply:
Sure. Sir.